README.md

```diff
@@ -55,7 +55,8 @@ Here we introduce the National Insurance contribution
            National Insurance contributions: £10.00
 
 Calculation rules:
- * National Insurance contributions: Any amount of money earned above a gross annual salary of £8,060.00 is subject to a National Insurance contribution of 12%
+ * the annual National Insurance (basic contributions): 12% of any amount of money earned above a gross annual salary of £8,060.00.
+ * "National Insurance contributions" on a salary slip is equal to the annual National Insurance (basic contributions) divided by 12.
```

Comment:
I think we should not do some assumptions that person will figure out what is the problem. I think I changed my mind after this interview and if we would like to assess ability to discover the domain probably we should plan the first part of the interview as a discovery session. If we want to assess only technical skills probably it will save some time to have more precise information how to calculate each field. 

I think that this is still not too technical description. We don't impose any specific of the code design. We just express the way it should be counted.